src/Page.test.jsx

```diff
@@ -0,0 +1,32 @@
+import { render, fireEvent } from '@testing-library/react';
+
+import Page from './Page';
+
+test('Page props로 전달한 이벤트 리스너가 정상작동하는지 확인하고, 전달된 테스크 목록이 보여지는지 확인한다.', () => {
+  const taskTitle = '';
+  const handleChange = jest.fn();
+  const handleClickAddTask = jest.fn();
+  const tasks = [{ id: 1, title: '기상' }];
+  const handleClickDeleteTask = jest.fn();
```

Comment:
```suggestion
  const taskTitle = '';
  const tasks = [{ id: 1, title: '기상' }];
  
  const handleChange = jest.fn();
  const handleClickAddTask = jest.fn();
  const handleClickDeleteTask = jest.fn();
```

의미상으로 구분되는 경우 빈 줄을 넣어 구분해주면 좋아요.
비슷한 변수끼리도 모아주면 코드 가독성이 더 좋을 거에요 ;)